noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/note.nr

```diff
@@ -218,14 +218,14 @@ fn compute_note_plaintext_for_this_strategy<Note, let N: u32>(
     storage_slot: Field,
 ) -> [u8; N * 32 + 64]
 where
-    Note: NoteInterface + Packable<N>,
+    Note: NoteType + Packable<N>,
```

Comment:
Been thinking for a while now that this should be `NOTE`, not `Note`. `Note` looks like a type, and we always use caps for generics in other places.

Reply:
e.g. below `Note::get_id()` looks quite confusing - as if `Note` was a trait, which also sounds quite reasonable.

Reply:
Agree. Created [an issue](https://github.com/AztecProtocol/aztec-packages/issues/12260) and I will tackle it once we get our note-related PRs merged.